hack/start.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+# Operator startup script, mainly for e2e tests.
+# - Applies everything from manifests/ directory
+# - Prepares operator Deployment template from hack/start-manifests based on $IMAGE_FORMAT (if set)
+# - Applies the Deployment
+#
+
+set -e
+
+usage() {
+    cat <<EOF
+$0 [-d]
+
+    -d: dry-run
+EOF
+    exit
+}
+
+dry_run=false
+
+while getopts ":hd" opt; do
+  case ${opt} in
+    h ) usage
+        ;;
+    d )
+        dry_run=true
+        if [ -z "$ARTIFACT_DIR" ]; then
+            echo 'WARNING: -d option has no effect without $ARTIFACT_DIR set'
+        fi
+        ;;
+    \? ) usage
+        ;;
+  esac
+done
+
+manifest=$(mktemp -d)
+
+cleanup(){
+    local return_code="$?"
+
+    set +e
+
+    # Save manifests for debugging if ARTIFACT_DIR is set.
+    if [ -n "$ARTIFACT_DIR" ]; then
+        mkdir -p $ARTIFACT_DIR/manifest
+        cp ${manifest}/* $ARTIFACT_DIR/manifest/
+    fi
+
+    rm -rf ${manifest}
+    exit $return_code
+}
+
+trap cleanup exit
+
+# Interpret $IMAGE_FORMAT. Following the same format as in
+# See https://github.com/openshift/origin/blob/master/test/extended/testdata/csi/README.md
+if [ -z "${IMAGE_FORMAT}" ] ; then
+    imageBasePrefix="registry.svc.ci.openshift.org/ocp/4.5:"
+    imageBaseSuffix=""
+else
+    imageBasePrefix=$( echo ${IMAGE_FORMAT} | sed -e 's/${component}.*//' )
+    imageBaseSuffix=$( echo ${IMAGE_FORMAT} | sed -e 's/.*${component}//' )
+fi
+
+cat <<EOF >${manifest}/.sedscript
+s,{{.AttacherImage}},${imageBasePrefix}csi-external-attacher${imageBaseSuffix},
+s,{{.ProvisionerImage}},${imageBasePrefix}csi-external-provisioner${imageBaseSuffix},
+s,{{.ResizerImage}},${imageBasePrefix}csi-external-resizer${imageBaseSuffix},
+s,{{.SnapshotterImage}},${imageBasePrefix}csi-external-snapshotter${imageBaseSuffix},
+s,{{.NodeDriverRegistrarImage}},${imageBasePrefix}csi-node-driver-registrar${imageBaseSuffix},
+s,{{.LivenessProbeImage}},${imageBasePrefix}csi-livenessprobe${imageBaseSuffix},
+s,{{.ImageBasePrefix}},${imageBasePrefix},
+s,{{.ImageBaseSuffix}},${imageBaseSuffix},
+EOF
+
+repo_dir="$(dirname $0)/.."
+
+cp -v ${repo_dir}/manifests/*.yaml ${manifest}/
```

Comment:
IMO, we should generate yaml files for local driver deployment from CSV in the future. Since we don't have it, I put the yaml files directly to `manifest/`. I hope it's just a temporary measure.